modules/articles/tests/server/article.server.routes.tests.js

```diff
@@ -306,7 +306,10 @@ describe('Article CRUD tests', function () {
     var articleObj = new Article(article);
 
     // Save the article
-    articleObj.save(function () {
+    articleObj.save(function (err) {
+      if (err) {
```

Comment:
My intent was only to make diagnosing test failures easier when the failure is related to the article not being saved.  When that save fails, the definitely fails, just not in a clear way.  

Nothing crazy, but dealing with that was a enough of a hassle at the time to motivate me to submit this patch.